### PR TITLE
[Loi Card] Highlight the card border on select

### PR DESCRIPTION
**Pull request by shobhitagarwal1612**
Towards #1153

* Converts CardView to MaterialCardView
* Fix incorrect tint color of the drawer icon
* Highlights the last selected card

https://user-images.githubusercontent.com/8918023/192594172-2604de73-f087-46fd-bc89-89147dd3cb2b.mp4

@JSunde  PTAL?

### CODE REVIEW

**Comment by JSunde** [comment]
Can you add a screenshot?

**Review by JSunde — APPROVED** [review]

File: ground/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -21,7 +21,7 @@
   <style name="AppTheme" parent="AppTheme.Base" />
 
   <style name="AppTheme.Launcher">
-    <item name="android:windowBackground">@drawable/launch_screen</item>
+<!--    <item name="android:windowBackground">@drawable/launch_screen</item>-->
```
Comment (JSunde) [inline ground/src/main/res/values/styles.xml:24]: Remove?

Comment (shobhitagarwal1612) [reply ground/src/main/res/values/styles.xml]: Done

**Comment by shobhitagarwal1612** [comment]
> Can you add a screenshot? 

Add to description

**Pull request merged**